File: hr.python.answer/laborer.py
class Laborer:
    def __init__(self, laborerId, laborerName, hireDate):
        self.laborerId = laborerId
        self.laborerName = laborerName
        self.hireDate = hireDate
    
    def getLaborerId(self):
        return self.laborerId
    
    def getLaborerName(self):
        return self.laborerName

    def getHireDate(self):
        return self.hireDate
    
    def setLaborerId(self, laborerId):
        self.laborerId = laborerId
    
    def setLaborerName(self, laborerName):
        self.laborerName = laborerName

    def setHireDate(self, hireDate):
        self.hireDate = hireDate
    
    def __str__(self):
        return f'{self.laborerId:>3}    {self.laborerName:<5}   {self.hireDate}'

File: 김다훈.hr.java/src/com/my/hr/domain/Laborer.java
package com.my.hr.domain;

public class Laborer {
	private int laborerId;
	private String laborerName;
	private String startDate;
	
	public Laborer(String laborerName, String startDate) {
		this.laborerName = laborerName;
		this.startDate = startDate;
	}
	
	public void setLaborerId(int laborerId) {
		this.laborerId = laborerId;
	}
	
	public int getLaborerId() {
		return this.laborerId;
	}
	
	public String getLaborerName() {
		return this.laborerName;
	}
	
	public void setLaborerName(String laborerName) {
		this.laborerName = laborerName;
	}
	
	public String getStartDate() {
		return this.startDate;
	}
	
	public void setStartDate(String startDate) {
		this.startDate = startDate;
	}
	
	@Override
	public String toString() {
		return String.format("%5d %8s %9s", 
				 laborerId, laborerName, startDate);
	}
}
